process-compose.yaml

```diff
@@ -119,6 +124,7 @@ processes:
       - ESPRESSO_SEQUENCER_PRIVATE_STAKING_KEY=$ESPRESSO_DEMO_SEQUENCER_STAKING_PRIVATE_KEY_1
       - ESPRESSO_SEQUENCER_PRIVATE_STATE_KEY=$ESPRESSO_DEMO_SEQUENCER_STATE_PRIVATE_KEY_1
       - ESPRESSO_SEQUENCER_ETH_ACCOUNT_INDEX=11
+      - ESPRESSO_SEQUENCER_STAKE_TABLE_CAPACITY=10
```

Comment:
are there any way to define a global (to all sequencer docker) env var declaration, so that we can centrally manged,
in case we forget to update one of them in the future?